othello/moderator/runners.py

```diff
@@ -0,0 +1,167 @@
+import os
+import select
+import signal
+import subprocess
+import time
+
+import psutil
+
+from django.conf import settings
+
+from ..sandboxing import get_sandbox_args
+from .moderator import Player
+from .utils import ServerError, UserError, capture_generator_value
+
+LEGACY_MOVES = {
+    (
+        x + 11
+        if x < 8
+        else x + 13
+        if x < 16
+        else x + 15
+        if x < 24
+        else x + 17
+        if x < 32
+        else x + 19
+        if x < 40
+        else x + 21
+        if x < 48
+        else x + 23
+        if x < 56
+        else x + 25
+    ): x
+    for x in range(64)
+}
+
+
+def legacy_board_convert(board):
+    legacy_board = ""
+    for i in range(100):
+        if i in LEGACY_MOVES:
+            legacy_board += board[LEGACY_MOVES[i]]
+        else:
+            legacy_board += "?"
+    return legacy_board.replace(Player.BLACK.value, "@")
```

Comment:
```suggestion
    table = {ord(Player.WHITE.value): "o", ord(player.BLACK.value): "@"}
    return "?" * 11 + "??".join(board[i: i + 8].translate(table) for i in range(0, 64, 8)) + "?" * 11
```

Untested, so please audit yourself before applying.